doc/traits/release.rst

```diff
@@ -39,3 +52,4 @@ Example
       preprocess: 'finalize' # recommended
     release:
       nextversion: 'bump_minor'
+      release_callback: 'release_callback'  # script is relative to main repo root directory
```

Comment:
please do not call it script. how about just stating 'relative to main repository root'

in general, I do not think it is a good pattern to add lengthy comments to our examples

Reply:
Done